contrib/exec-healthz/Makefile

```diff
@@ -0,0 +1,18 @@
+all: push
+
+# 0.0 shouldn't clobber any released builds
+TAG = 0.0
+PREFIX = bprashanth/healthz-server
+
+server: simpleHealthzServer.go
+	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-w' -o healthz_server ./simpleHealthzServer.go
+
+container: server
+	docker build -t $(PREFIX):$(TAG) .
+
+push: container
+	# gcloud docker push gcr.io/google_containers/$(PREFIX):$(TAG)
```

Comment:
Reading gcr.io docs. Don't think I have the permissions to push to google_containers and I'm yet to setup my own registry, so it's just docker push for now.